docs/vectors/python/argondriver.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+#
+from __future__ import division, print_function
+
+import argparse
+import json
+import random
+import string
+import subprocess
+import sys
+
+
+class argonRunner(object):
```

Comment:
This is an awfully involved class for something that really just invokes a subprocess in a loop. Do we really need something this complex? Is it taken from somewhere else?

Reply:
This is the same code I've put on a gist at the time of argon2i reference vectors generation, to which I just added the option to generate argon2id and argon2d password hashes. At least, adding the other hash constructions has been a no-brainer